dashboard/test/models/course_test.rb

```diff
@@ -478,13 +478,12 @@ class ProgressTests < ActiveSupport::TestCase
   end
 
   test "valid_courses" do
-    # The data here must be in sync with the data in ScriptConstants.rb
-    csp = create(:course, name: 'csp-2017')
+    csp = create(:course, name: 'csp-2017', visible: true, is_stable: true)
```

Comment:
shouldn't need to specify is_stable to make things show up in valid_courses, but could be ok to include if you're testing the inclusion of the is_stable field in the valid_courses payload